### PR TITLE
build(aio): new migration

### PR DESCRIPTION
**Pull request by Foxandxss**
New migration. It contains the new changes of a `tools` folder with the boilerplate files.

### CODE REVIEW

**Review by wardbell — APPROVED** [review]
LGTM. All are content files. Big enough that we should have one other set of eyes.

File: aio/content/examples/universal/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+aot/**/*.ts
+**/*.ngfactory.ts
+**/*.ngsummary.json
+**/*.metadata.json
+**/*.js
+dist
+!app/tsconfig.json
+!/*.js
```
Comment (petebacondarwin) [inline aio/content/examples/universal/.gitignore]: weren't we going to use only top level gitignore?

File: aio/content/guide/appmodule.md [hunk 2]
```diff
@@ -14,12 +16,14 @@ The [setup](guide/setup) instructions produce a new project with the following m
 You'll evolve this module as your application grows.
 
 
-<code-example path="setup/src/app/app.module.ts" linenums="false">
+<code-example path="setup/src/app/app.module.ts" title="src/app/app.module.ts" linenums="false">
```
Comment (petebacondarwin) [inline aio/content/guide/appmodule.md]: loving the titles!

File: aio/content/guide/appmodule.md [hunk 3]
```diff
@@ -14,12 +16,14 @@ The [setup](guide/setup) instructions produce a new project with the following m
...
 
 </code-example>
 
+
+
 After the `import` statements, you come to a class adorned with the
-**`@NgModule`** [_decorator_](guide/glossary).
+**`@NgModule`** [_decorator_](guide/glossary#decorator '"Decorator" explained').
```
Comment (petebacondarwin) [inline aio/content/guide/appmodule.md]: and tooltip titles!

File: aio/content/guide/browser-support.md [hunk 4]
```diff
@@ -347,13 +369,15 @@ Here are the features which may require additional polyfills:
   </tr>
 
   <tr style="vertical-align: top">
-     
+
```
Comment (petebacondarwin) [inline aio/content/guide/browser-support.md]: note the removal of excess whitespace :-)

**Pull request merged**